<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDialogBatchRegister</class>
 <widget class="BaseDlg" name="QDialogBatchRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QDialogBatchRegister</string>
  </property>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>377</x>
     <y>258</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_button">
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>258</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="LineEdit" name="sample_stopno">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="LineEdit" name="sample_count">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="stop_sample_radio">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>261</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>末位样本号</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="sample_count_radio">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>311</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>样本数量</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>262</y>
     <width>321</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>项目重复次数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="assayRunTimes">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>shared/CReadOnlyDelegate.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
